public/app/core/components/sidemenu/TopSection.tsx

```diff
@@ -1,26 +1,37 @@
-import React, { FC } from 'react';
-import { cloneDeep, filter } from 'lodash';
-import TopSectionItem from './TopSectionItem';
-import config from '../../config';
+import React from 'react';
+import { cloneDeep } from 'lodash';
 import { locationService } from '@grafana/runtime';
+import { Icon, IconName } from '@grafana/ui';
+import SideMenuItem from './SideMenuItem';
+import config from '../../config';
+import { NavModelItem } from '@grafana/data';
 
-const TopSection: FC<any> = () => {
-  const navTree = cloneDeep(config.bootData.navTree);
-  const mainLinks = filter(navTree, (item) => !item.hideFromMenu);
-  const searchLink = {
-    text: 'Search dashboards',
-    icon: 'search',
-  };
+const TopSection = () => {
+  const navTree: NavModelItem[] = cloneDeep(config.bootData.navTree);
+  const mainLinks = navTree.filter((item) => !item.hideFromMenu);
 
   const onOpenSearch = () => {
     locationService.partial({ search: 'open' });
   };
 
   return (
     <div data-testid="top-section-items" className="sidemenu__top">
-      <TopSectionItem link={searchLink} onClick={onOpenSearch} />
+      <SideMenuItem label="Search dashboards" onClick={onOpenSearch}>
+        <Icon name="search" size="xl" />
+      </SideMenuItem>
       {mainLinks.map((link, index) => {
-        return <TopSectionItem link={link} key={`${link.id}-${index}`} />;
+        return (
+          <SideMenuItem
+            key={`${link.id}-${index}`}
+            label={link.text}
+            menuItems={link.children}
+            target={link.target}
+            url={link.url}
+          >
+            {link.icon && <Icon name={link.icon as IconName} size="xl" />}
+            {link.img && <img src={link.img} />}
```

Comment:
☑️ To make this `img` accessible maybe we could add the `alt` property with the combination of the `${link.text} logo`? I think we get those images for plugins app

Reply:
I think this is related to screen readers using the mouse on hover or something, here the [guideline](https://dequeuniversity.com/rules/axe/4.3/image-alt?application=AxeChrome) :), about tackling in other PR sure :), for now, we don't have the Pa11y CI blocking the merge on PR stage :)